src/server.h

```diff
@@ -198,10 +198,10 @@ class Server {
 
   Stats stats_;
   Engine::Storage *storage_;
-  Cluster *cluster_;
+  std::unique_ptr<Cluster> cluster_;
   static std::atomic<int> unix_time_;
-  class SlotMigrate *slot_migrate_ = nullptr;
-  class SlotImport *slot_import_ = nullptr;
+  std::unique_ptr<class SlotMigrate> slot_migrate_;
+  class SlotImport *slot_import_;
```

Comment:
```suggestion
  class SlotImport *slot_import_ = nullptr;
```
fix the cppcheck warning in CI.